sklearn/metrics/tests/test_classification.py

```diff
@@ -1600,6 +1605,25 @@ def test_log_loss():
     assert_almost_equal(loss, 1.0630345, decimal=6)
 
 
+@pytest.mark.parametrize(
+    'y_true, y_proba, labels',
+    [(MultiLabelBinarizer().fit_transform(
+        [['cat', 'dog', 'unicorn'], ['cat'], ['dog', 'unicorn'],
+         ['cat', 'unicorn']]),
+      np.asarray([[.9, .1, .1], [.6, .8, .1], [.7, .6, .8], [.3, .2, .8]]),
+      None),
+     (np.ones((1, 3)), .5 * np.ones((1, 3)), [0, 1])])
```

Comment:
Hmmm... Labels should perhaps not work like this for multilabel. For example, in precision_score, labels can be used to select columns of multilabel data for inclusion in the average. Not sure what to do about that. But we may eventually want labels to support the multi-output multiclass case. Perhaps specifying labels with multilabel data should raise NotImplementedError for now

Reply:
Hi @jnothman , the labels in the test is only for binary log loss case when y_true contains only 1 label. for multi-label case, I don't think we actually need ``labels=`` at least right now.